Gems/EMotionFX/Code/Tests/Mocks/CommandManager.h

```diff
@@ -11,7 +11,7 @@ namespace MCore
     class CommandManager
     {
     public:
-        //virtual ~CommandManager();
+        virtual ~CommandManager() = default;
 
         //bool ExecuteCommand(const char* command, AZStd::string& outCommandResult, bool addToHistory = true, Command** outExecutedCommand = nullptr, CommandLine* outExecutedParamters = nullptr, bool callFromCommandGroup = false, bool clearErrors = true, bool handleErrors = true);
```

Comment:
ugh, why are all the methods just commented out?

Reply:
I think they copied the original CommandManager and just enabled the ones required by the tests... 
Leaving any fix to this out of this PR